.travis.yml

```diff
@@ -68,13 +69,11 @@ install:
   - "travis_retry pip install -e .[${EXTRAS}]"
 
 before_script:
-  - "export DISPLAY=:99.0" #set up xvfb, req'd for non-phantomJS webdrivers
-  - "sh -e /etc/init.d/xvfb start"
-  - "export E2E_WEBDRIVER_BROWSERS='Firefox'"
-  - "export E2E_WEBDRIVER_TIMEOUT='300'" # Max allowed time for E2E tests
+  - "if [ ${E2E} == 'yes' ]; then export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start; export E2E_WEBDRIVER_BROWSERS='Firefox'; export E2E_WEBDRIVER_TIMEOUT='300'; fi"
```

Comment:
I did `+x` - the script was executed, but the E2E tests failed with the
same error you'd get if the commands had not been executed.

On Wed, Jan 20, 2016, 11:35 Jiri Kuncar <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/inveniosoftware/invenio-accounts/pull/88#discussion_r50238379>
> :
>
> > @@ -68,13 +69,11 @@ install:
> >    - "travis_retry pip install -e .[${EXTRAS}]"
> >
> >  before_script:
> > -  - "export DISPLAY=:99.0" #set up xvfb, req'd for non-phantomJS webdrivers
> > -  - "sh -e /etc/init.d/xvfb start"
> > -  - "export E2E_WEBDRIVER_BROWSERS='Firefox'"
> > -  - "export E2E_WEBDRIVER_TIMEOUT='300'" # Max allowed time for E2E tests
> > +  - "if [ ${E2E} == 'yes' ]; then export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start; export E2E_WEBDRIVER_BROWSERS='Firefox'; export E2E_WEBDRIVER_TIMEOUT='300'; fi"
>
> did you chmod u+x e2e_setup.sh in git?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inveniosoftware/invenio-accounts/pull/88/files#r50238379>
> .
>